patterns/2-structured/repository-activity-score.md

```diff
@@ -58,15 +58,15 @@ function calculateScore(repo) {
     // initial score is 50 to give active repos with low GitHub KPIs (forks, watchers, stars) a better starting point
     let iScore = 50;
     // weighting: forks and watches count most, then stars, add some little score for open issues, too
-    iScore += repo["forks_count"] * 5 + repo["watchers_count"] + repo["stargazers_count"] / 3 + repo["open_issues_count"] / 5;
+    iScore += repo.forks_count * 5 + (repo.subscribers_count ? repo.subscribers_count : 0) + repo.stargazers_count / 3 + repo.open_issues_count / 5;
```

Comment:
```suggestion
    iScore += repo.forks_count * 5;
    iScore += (repo.subscribers_count ? repo.subscribers_count : 0)
    iScore += repo.stargazers_count / 3;
    iScore += repo.open_issues_count / 5;
```

Now that I am readying this with fresh eyes, I am wondering if it would be easier to understood if we break up this calculation into multiple steps? This way it would be more obvious how each of the properties is weighted. What do you think?